### ISSUE TITLE
Let user specify the stream to print messages

### ISSUE THREAD

**Issue by yifan-gu** [appc/docker2aci#61]
Currently whether the message is printed to stdout or stderr is hardcoded in the lib, we want to have user be able to specify the stream.
Also see #60 .

**Comment by s-urbaniak** [comment]
This is addressed in #218 where the user can specify an arbitrary (stdlib compatible) logger in `CommonConfig`:

```
 type CommonConfig struct {
 ...
 	Info  log.Logger
 	Debug log.Logger
 }
```

where

```
type Logger interface {
	Print(...interface{})
	Printf(string, ...interface{})
	Println(...interface{})
}
```

So it is up to the user to use a stdlib Logger using https://godoc.org/log#New, or implement the interface itself.

**Comment by lucab** [comment]
Fixed in #218.
